### DISCUSSION TITLE
Twitter and X confusion

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#396] [General]
I think the renaming of Twitter to X generated some confusion in the theme. Same parts of the theme use the twitter variable, while others use x.

I get this error:

`WARN  [icons] simple-icons's twitter doesn't exist.`

For [example](https://pushevs.com/authors/maximilianholland/), the authors page already accepts the x variable, but shows an invalid link.

To make the theme work, I have to use x sometimes, and others twitter, check my [params.yaml](https://github.com/LITUATUI/pushevs.com/blob/main/config/_default/params.yaml)

Maybe it's time to move everything to x and no longer recognize twitter as a valid variable.

**Comment by razonyang** [comment]
Should be fixed by [github.com/hbstack/socials@v0.16.0](https://github.com/hbstack/socials/releases/tag/v0.16.0).

> Maybe it's time to move everything to x and no longer recognize twitter as a valid variable.

For backward compatibility, I keep the Twitter, but use X icon as Twitter's, you can use `x` now as well, `x` is recommended, `twitter` may be remove from HB themes in the future.

**Comment by LITUATUI** [comment]
What about share buttons?

`WARN  [social-share-buttons] unsupported media: x`

**Reply by razonyang** [reply]
Aha, I forgot this, should be fixed by [github.com/hbstack/social-share-buttons@v0.4.0](https://github.com/hbstack/social-share-buttons/releases/tag/v0.4.0), but I didn't test, please test and confirm if it work, since my VPN server is down for some reason, could not  access X.

**Reply by LITUATUI** [reply]
Everything is OK now, thanks, you're the best!
